src/platform/forms-system/src/js/definitions/directDeposit.jsx

```diff
@@ -139,6 +113,11 @@ const uiSchema = ({ affectedBenefits, unaffectedBenefits, optionalFields }) => {
   return ui;
 };
 
+/**
+ * Constructs schema in "sections" to perserve order since the Review page doesn't like view properties
```

Comment:
What does "the Review page doesn't like view properties" mean? We have `view` properties in other forms that don't seem to be causing any problems on the review page.